catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/TeamRepository.java

```diff
@@ -46,7 +52,7 @@ public TeamRepository(CollectionDAO dao) {
         TEAM_PATCH_FIELDS,
         TEAM_UPDATE_FIELDS,
         false,
-        false,
+        true,
```

Comment:
This will also enable the owner based unit tests from the EntityRepositoryTest class